src/GLFWHelper.cpp

```diff
@@ -85,6 +85,9 @@ int initializeGLFW(GLFWwindow **window) {
 	// Set keyboard callback.
 	glfwSetKeyCallback(*window, key_callback);
 
+   // Set sticky keys for more accurate polling
+   glfwSetInputMode(*window, GLFW_STICKY_KEYS, 1);

```

Comment:
This ensures that polling keys doesn't miss things. For instance without sticky keys if I hit `w` really fast and release and then poll for the state of `w` it wouldn't be `GLFW_PRESS`. With sticky keys on the state of any key (if pressed at any point) will be `GLFW_PRESS` until the key is polled.

From the docs:
> When sticky keys mode is enabled, the pollable state of a key will remain GLFW_PRESS until the state of that key is polled with glfwGetKey. Once it has been polled, if a key release event had been processed in the meantime, the state will reset to GLFW_RELEASE, otherwise it will remain GLFW_PRESS.